cypress/integration/v2/blocks/card.spec.js

```diff
@@ -67,6 +69,28 @@ function switchDesign() {
 	] ) )
 }
 
+function typeContent() {
+	it( 'should allow typing in the block', () => {
+		cy.setupWP()
+		cy.newPage()
+		cy.addBlock( 'ugb/card' ).as( 'cardBlock' )
+		registerBlockSnapshots( 'cardBlock' )
+
+		cy.openInspector( 'ugb/card', 'Style' )
+		cy.collapse( 'General' )
+		cy.adjust( 'Columns', 1 )
```

Comment:
Maybe it's better if no adjustments are made at the start. If you need to remove the other columns because `typeBlock` and `assertBlockContent` matches multiple elements, maybe those functions need to be adjusted to only perform the action on just the first matching selector.

Reply:
Okay got it! I'll adjust them to handle only a single element